packages/line/src/props.js

```diff
@@ -64,8 +64,14 @@ export const LinePropTypes = {
 
     enableGridX: PropTypes.bool.isRequired,
     enableGridY: PropTypes.bool.isRequired,
-    gridXValues: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
-    gridYValues: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
+    gridXValues: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.number, PropTypes.string])),
```

Comment:
The line chart also supports Date objects, so you should add `PropTypes.instanceOf(Date)` to both `gridXValues` and `gridYValues`.

Reply:
Thanks for the feedback, I wasn't sure about that. I've added it now.